apps/omg_performance/test/omg_performance/integration/bazantine_event_test.exs

```diff
@@ -0,0 +1,145 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Performance.BazantineEventsTest do
+  use ExUnitFixtures
+  use ExUnit.Case, async: false
+  use OMG.ChildChain.Integration.Fixtures
+  use OMG.Watcher.Fixtures
+  alias OMG.Eth
+  require OMG.Utxo
+
+  alias OMG.Eth
+  alias OMG.Eth.RootChain
+  alias OMG.Performance
+  alias OMG.State.Transaction
+  alias OMG.Utxo
+  alias OMG.Watcher.HttpRPC.Client
+  alias OMG.Watcher.TestHelper
+
+  @moduletag :integration
+  @moduletag timeout: 70_000
+
+  @tag fixtures: [:watcher, :child_chain, :contract]
+  test "time response for asking for exit data", %{contract: %{contract_addr: contract}} do
```

Comment:
All other concerns aside, I'm missing a documented way to actually run this performance test, like we have with the old perf test. A test put in `omg_performance/test/` should only test that the perf test runs fine, and not be the way to invoke the perf test

Reply:
after add more workers, will expand the demo_05 and expand ByzantineEvents documentation